src/page/bank-payment.page.ts

```diff
@@ -1,11 +1,11 @@
-import { $, ElementFinder } from 'protractor';
+import { ElementFinder, element, by } from 'protractor';
 
 export class BankPaymentPage {
   private confirmOrderBtn: ElementFinder;
 
   constructor () {
     // we can remove span
-    this.confirmOrderBtn = $('#cart_navigation button');
+    this.confirmOrderBtn = element(by.cssContainingText('span', 'I confirm my order'));
```

Comment:
👍

Reply:
Another option was: by.partialButtonText('I confirm my order') but, it yours also is good